interfaces/Portal/src/app/services/filter.service.ts

```diff
@@ -35,6 +36,20 @@ export enum FilterOperatorEnum {
   providedIn: 'root',
 })
 export class FilterService {
+  public DEFAULT_FILTER_OPTION: Filter = {
+    name: 'quickSearch',
```

Comment:
Will this string-value be also used as a query-parameter in the request? Then we can make it 'the same' by using an enum-value somewhere..

Are other query-parameters already somewhere defined in an enum?
Somewhere near `FilterOperatorEnum`?